src/move_basic.cpp

```diff
@@ -725,6 +736,25 @@ bool MoveBasic::moveLinear(tf2::Transform& goalInDriving,
         remaining = goalInBase.getOrigin();
         double distRemaining = sqrt(linear.x() * linear.x() + linear.y() * linear.y());
 
+// 
+	if (distRemaining < prevDistance) {
+		double current = ros::Time::now().toSec();
+		if (current-last > abortTimeout) {
+			abortGoal("MoveBasic: Goal update timeout.");		
```

Comment:
A more detailed abort message would be useful.

"No progress towards goal for longer than timeout."